add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1348,17 +1348,26 @@ this.MozLoopService = {
     LoopRooms.on("update", onRoomsChange);
     LoopRooms.on("delete", onRoomsChange);
     LoopRooms.on("joined", (e, room, participant) => {
+      // The participant that joined isn't necessarily included in room.participants (depending on
+      // when the broadcast happens) so concatenate.
+      let isOwnerInRoom = room.participants.concat(participant).some(p => p.owner);
+      let window = gWM.getMostRecentWindow("navigator:browser");
+      if (participant.owner) {
+        // is joining call and determine if other is here
+        let participantRoomCount = room.participants.concat(participant).length;
+        window.LoopUI._maybeShowBrowserSharingInfoBar(participantRoomCount > 1);
+      } else {
+        if (isOwnerInRoom) {
```

Comment:
Please do the simplification if just computing participant room count at the top, and don't check the owner.